test-packages/type-tests/test/mail-client/mail-client.spec.ts

```diff
@@ -6,9 +6,16 @@ const mailConfig = { from: 'from', to: 'to' };
 sendMail({ destinationName: 'dest' }, mailConfig);
 // $ExpectType Promise<MailResponse[]>
 sendMail({ destinationName: 'dest' }, [mailConfig, mailConfig], {
-  parallel: true
+  tls: {
+    rejectUnauthorized: false
+  }
 });
 // $ExpectType Promise<MailResponse[]>
-sendMail({ destinationName: 'dest' }, [mailConfig, mailConfig], {
-  parallel: false
-});
+sendMail(
+  { destinationName: 'dest' },
+  [mailConfig, mailConfig],
+  {},
```

Comment:
This might be a concern, as we now have a new optional option.

We can also merge the following into one optional parameter:
- MailClientOptions
- SmtpTransportOptions

However, as the SmtpTransportOptions has to sync with the lib (out of our control), we might have key conflicts theoretically. Therefore, I would then rename the `parallel` key and make it unique like { sdk: { parallel: true }}.

Reply:
ah, I was leaving a comment before I read this. I think having a `sapCloudSdkOptions` or so is a good idea to avoud conflicts.